src/widgets/histogram/content-view.js

```diff
@@ -213,6 +217,18 @@ module.exports = cdb.core.View.extend({
     this.model.bind('change:avg', this._onChangeAvg, this);
   },
 
+  _unbinds: function () {
+    this._dataviewModel.off('change:bins', this._onChangeBins, this);
```

Comment:
Because we are calling `render` when collapsing, this function unbinds events in order to not duplicate bindings. Below `_setupBindings` is called again.